app/controllers/unit_communications_controller.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class UnitCommunicationsController < ApplicationController
+  def index
+    head :unauthorized unless current_user.role_unit_communication?
```

Comment:
Ich glaube ich würde es doch in den Units-Controller einpflanzen, dann geht das mit cancan einfacher:

```
# in app/controllers/units_controller,rb

def emails
    @units = @units.where(id: params[:unit_ids]&.split(','))
    @emails = @units.map { |unit| unit.lagerleiter.email }
end
```

- ```authorize :email, Unit``` wird dann automatisch gemacht
- Ich würde unit_ids direkt als Parameter übergeben
- Ich würde die emails erst in der View joinen